pkg/volume/cinder/attacher.go

```diff
@@ -75,7 +75,7 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 
 	pending, volumeStatus, err := attacher.cinderProvider.OperationPending(volumeID)
 	if err != nil {
-		return "", err
+		// Continue attach, the same as DiskIsAttached()
```

Comment:
I see, I should return err when OperationPending() or DiskIsAttached() return err.
The err will be 'Unable to initialize cinder client', return err better.